ui/DesignSystem/Component/Wallet/SendReceive.svelte

```diff
@@ -1,10 +1,26 @@
 <script>
+  import { push } from "svelte-spa-router";
+  import * as path from "../../../src/path.ts";
+  import {
+    payerStore,
+    recipientStore,
+    amountStore,
+  } from "../../../src/transfer.ts";
+
   import { Button, Input, Icon, Title } from "../../Primitive";
   import Receive from "./Receive.svelte";
 
   export let accountId = null;
+  let recipient,
+    amount,
+    currentlyActiveSend = true;
 
-  $: currentlyActiveSend = true;
+  const openSendModal = () => {
```

Comment:
There are two ways to do store updates with forms:

You can `set` the `store` upon form submission, so nothing is saved until the user decides to click a submit button. That's what is going on here - the store doesn't change until `openSendModal` is called.

You can `set` the `store` reactively every time the user changes the input value, as seen below in `SendFunds`:
```
$: amountStore.set(amount); 
```
With this approach, everything is automatically saved while the user types - see [docs on reactive declarations here](https://svelte.dev/tutorial/reactive-declarations).

Both are valid and have their pros and cons; it's a question of UX.

Reply:
For this form, I would go for option 1 as at this stage the store is set only to pass on the values to the modal. No complex validations happen here. In the modal the store is set reactively.